.swiftlint.yml

```diff
@@ -1,110 +1,106 @@
+#   - trailing_comma
+#   - trailing_newline
+#   - trailing_semicolon
+#   - trailing_whitespace
+#   - type_body_length
+#   - type_name
+#   - unused_closure_parameter
+#   - unused_enumerated
+#   - unused_optional_binding
+#   - valid_docs
+#   - valid_ibinspectable
+#   - vertical_parameter_alignment
+#   - vertical_whitespace
+#   - weak_delegate
 
-  - todo
-  - variable_name
 opt_in_rules:
-  - closure_spacing
-  - empty_count
   - explicit_init
-  - force_unwrapping
+  - fatal_error_message
+  - number_separator
   - overridden_super_call
-  - private_outlet
+  - prohibited_super_call
+  - redundant_void_return
+  - sorted_imports
+
+disabled_rules:
+  - conditional_returns_on_newline
+  - cyclomatic_complexity
+  - empty_count
+  - explicit_type_interface
+  - file_header
+  - for_where
+  - function_body_length
+  - function_parameter_count
+  - large_tuple
+  - line_length
+  - nimble_operator
+  - notification_center_detachment
+  - shorthand_operator
   - switch_case_on_newline
-included: # paths to include during linting.
-excluded: # paths to ignore during linting. Takes precedence over `included`.
-  - Carthage
-  - Pods
-  - Source/ExcludedFolder
-  - Source/ExcludedFile.swift
+  - todo
+  - void_return
 
-# Rule configuration
-closing_brace: error
-closure_spacing: error
-colon: error
-comma: error
-conditional_returns_on_newline: warning
-control_statement: error
-cyclomatic_complexity:
-  warning: 8
-  error: 20
-empty_count: error
-explicit_init: error
-file_length:
-  warning: 500
-  error: 1000
-force_cast: error
-force_try: error
-force_unwrapping: error
-function_body_length:
-  warning: 40
-  error: 100
-function_parameter_count:
-  warning: 6
-  error: 8
-leading_whitespace: error
-legacy_cggeometry_functions: error
-legacy_constant: error
-legacy_constructor: error
-legacy_nsgeometry_functions: error
-line_length:
-  warning: 140
-  error: 200
-mark: error
-missing_docs: warning
-nesting: error
-opening_brace: error
-operator_whitespace: warning
-overridden_super_call:
-  severity: warning
-  included: default
-  exclued: default
-private_outlet:
-  severity: warning
-  allow_private_set: false
-private_unit_test:
-  severity: error
-  regex: "XCTestCase"
-redundant_nil_coalesing: error
-return_arrow_whitespace: error
-statement_position:
-  statement_mode: default
-  severity: error
-switch_case_on_newline: error
-todo: warning
-trailing_newline: error
-trailing_semicolon: error
-trailing_whitespace:
-  severity: error
-  ignores_empty_lines: false
-type_body_length:
-  warning: 300
-  error: 500
-type_name:
-  min_length:
-    warning: 3
-    error: 0
-  max_length:
-    warning: 50
-    error: 1000
-valid_docs: warning
-valid_ibinspectable: error
-variable_name:
-  min_length:
-    warning: 3
-    error: 0
-  max_length:
-    warning: 50
-    error: 1000
-vertical_whitespace: warning
+## Rule Configuration
 
-reporter: "xcode" # reporter type (xcode, json, csv, checkstyle, junit)
+apply_to_dictionaries: false
```

Comment:
I think this rule is configured incorrectly.  This should be a sublevel of `colon:`